### DISCUSSION TITLE
At&T_Sony Xperia 1 V

### DISCUSSION THREAD

**Discussion by ServusServorumDei** [kyujin-cho/pixel-volte-patch#224] [Carrier-specific Experiences]
@kyujin-cho 

Sony Xperia 1 V, Wifi calling stopped working after Android 14 update. I can verify VoNR and VoLTE working. I need VoWiFi calling function as I work Offshore. Is there a way I can make a dump so you can look this over.

I followed the VoLTE repair with VoWiFi.

**Comment by ServusServorumDei** [comment]
@kyujin-cho nfo shows VoWiFi as provisioned and sliders for VoWiFi is on

**Reply by fillwithjoy1** [reply]
Send screenshot of IMS status (from `*#*#4636#*#*`)

**Comment by fillwithjoy1** [comment]
Your issue might be due to Sony itself (see #251)
